playbook/app/pb_kits/playbook/pb_date_picker/docs/_date_picker_flatpickr_methods.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= pb_rails("button", props: { id: "close-btn", text: "Close" }) %>
+<%= pb_rails("button", props: { id: "clear-btn", text: "Clear", variant: "secondary" }) %>
+<%= pb_rails("button", props: { id: "today-btn", text: "Today" }) %>
```

Comment:
Can you use spacing props on the middle one to add a margin between? Can you make all the buttons the same color?